samples/text/text.cpp

```diff
@@ -1204,6 +1207,14 @@ MyPanel::MyPanel( wxFrame *frame, int x, int y, int w, int h )
       wxPoint(180,170), wxSize(200,70), wxTE_MULTILINE | wxTE_PROCESS_ENTER );
     m_enter->SetClientData((void *)wxT("enter"));
 
+#if wxUSE_SPELLCHECK    
+    m_spell = new MyTextCtrl(this, 100, wxT("Multiline, spellcheck enabled."),
+    wxPoint(180,270), wxSize(200,70), wxTE_MULTILINE);
+    wxASSERT(m_spell->EnableSpellcheck(true) != wxTEXT_SPELLCHECK_ERROR);
+    wxASSERT(m_spell->IsSpellcheckEnabled());
```

Comment:
Does this really need 2 asserts?

Reply:
No, this is here only for troubleshooting during development.

Reply:
Then don't make it part of the pull request.